spec/models/element_spec.rb

```diff
@@ -47,9 +47,12 @@ module Alchemy
         expect(subject.contents.count).to eq(element.contents.count)
       end
 
-      it "should create a new record with all attributes of source except given differences" do
-        copy = Element.copy(element, {name: 'foobar'})
-        expect(copy.name).to eq('foobar')
+      context 'with differences' do
+        subject(:copy) { Element.copy(element, {name: 'foobar'}) }
+
+        it "should create a new record with all attributes of source except given differences" do
+          expect(copy.name).to eq('foobar')
```

Comment:
The description says
> ...with all attributes of source except...

But you are testing only the difference.

If there is a test covering this already its maybe better to just change the description of the test like this `it "creates a new record where the given attributes differ from the origin" do ...`